frontend/src/landing/RestaurantTile.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useHistory } from "react-router";
+import GridList from "@material-ui/core/GridList";
+import GridListTile from "@material-ui/core/GridListTile";
+import GridListTileBar from "@material-ui/core/GridListTileBar";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardMedia from "@material-ui/core/CardMedia";
+import style from "./LandingPage.module.css";
+import changePath from "../general/helperFunctions";
+
+const RestaurantTile = (props) => {
+  const { setRestaurant } = props;
+  const history = useHistory();
+  const toBooking = (restaurant) => {
+    changePath("/booking", history);
+    setRestaurant(restaurant);
+  };
+  const fakeData = [
+    {
+      name: "NANDOZ",
+      times: [],
+      img: "",
+    },
+    {
+      name: "KCF",
+      times: [],
+      img: "",
+    },
+  ];
+
+  const columns = 3;
+  const cellHeight = 250;
+  // root height adjustment based on number of restaurants
+  const rootHeight = {
+    height: Math.ceil(fakeData.length / columns) * (cellHeight + 50),
+  };
+
+  return (
+    <div className={style.gridRoot} style={rootHeight}>
+      <GridList
+        cellHeight={cellHeight}
+        spacing={40}
+        className={style.gridList}
+        cols={columns}
+      >
+        {fakeData.map((data) => (
```

Comment:
Might want to think about loaders. When we load the restaurant data we have to wait for the API call so we want to show some indication of loading and some sign of failure if it doesn't load. Possibly <Suspense>

Reply:
>> Will do this  as another issue since back-end and front-end integration hasn't been done

Reply:
Definitely sounds like it should be part of a separate issue.